demos/dialog/index.html

```diff
@@ -153,6 +155,30 @@
         Secondary
       </mwc-button>
     </mwc-dialog>
+
+    <mwc-button data-num="8" raised>Form Validation</mwc-button>
+    <mwc-dialog id="dialog8" heading="Form Validation">
+      <p>We can validate user input before closing our dialog.</p>
+      <mwc-textfield
+        id="dialog8-text-field"
+        type="text"
+        pattern="^[a-zA-Z\s]{3,64}$"
```

Comment:
The `minlength` and `maxlength` attributes would be more appropriate than trying to encode them into the pattern.

Reply:
simpler too. i agree

Reply:
done